dss/api/bundles/__init__.py

```diff
@@ -265,11 +267,7 @@ def delete(uuid: str, replica: str, json_request_body: dict, version: str = None
     uuid = uuid.lower()
     tombstone_id = BundleTombstoneID(uuid=uuid, version=version)
     bundle_prefix = tombstone_id.to_key_prefix()
-    tombstone_object_data = _create_tombstone_data(
-        email=email,
-        reason=json_request_body.get('reason'),
-        version=version,
-    )
+    tombstone_object_data = Tombstone(email=email, reason=json_request_body.get('reason'))._asdict()
```

Comment:
Previously, the `version` kwarg was being handled, but is not with the refactor. Is there a reason `version` is now ignored?

Reply:
This follows the tombstone format outlined in the [deletion RFC](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0004-dss-deletion-process.md#deletion-process